src/main/resources/data/crimson/tags/blocks/logs.json

```diff
@@ -1,5 +1,5 @@
 {
- "values": [
-    "crimson:crimson_log"
- ]
+	"values": [
+		"#crimson:crimson_logs"
```

Comment:
2 spaces in json

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.